default.style

```diff
@@ -24,7 +24,9 @@
 #
 # linear - Create a column for this tag
 #
-# phstore - Don't create a column for this tag, but objects with the tag are areas
+# hstore - add tag to hstore only instead of its own column
```

Comment:
This is confusing, the flag rather does something along the lines of "don't create a separate column for this tag, but always include objects with this tag".